pytest_robotframework/_internal/robot/utils.py

```diff
@@ -241,6 +243,7 @@ def cli_defaults(settings_class: Callable[[dict[str, object]], _BaseSettings]) -
     # the cwd gets updated, robot will still run with the outdated cwd.
     # we set it in this wacky way to make sure it never overrides user preferences
     _BaseSettings._cli_opts["OutputDir"] = ("outputdir", ".")  # pyright:ignore[reportUnknownMemberType,reportPrivateUsage]
+    RobotSettings._extra_cli_opts["ConsoleType"] = ("console", "quiet")  # pyright:ignore[reportUnknownMemberType,reportPrivateUsage,reportArgumentType]
```

Comment:
maybe add a comment or update the comment above to explain this

Reply:
dome?

Reply:
Dear @KotlinIsland,

I noticed your recent comment referencing a "dome." Unfortunately, I am unable to understand what you are referring to in this context.

As we are using English as the primary language for this project, could you please clarify your comment and update it accordingly? This will ensure that all team members can fully comprehend the discussion.

Thank you for your cooperation.

Best regards,
[Your Name]